backend/src/Service/Subscription/SubscriptionService.php

```diff
@@ -53,26 +52,30 @@ public function getIsFutureState($storeOwner): INT
         if($subscriptionCurrent) {
             if($subscriptionCurrent->getStatus() === SubscriptionConstant::SUBSCRIBE_ACTIVE) {
              
-                return  1;
+                return  SubscriptionConstant::IS_FUTURE_TRUE;
             }
 
-            return  0;
+            return  SubscriptionConstant::IS_FUTURE_FALSE;
         }
 
-        return  0;
+        return  SubscriptionConstant::IS_FUTURE_FALSE;
     }
 
     /**
      * @param storeOwner
      * @return RemainingOrdersResponse|string
      */
-    public function packageBalance($storeOwner):RemainingOrdersResponse|string
+    public function packageBalance($storeOwner): RemainingOrdersResponse|string
     {
        $this->checkSubscription($storeOwner);
       
        $subscription = $this->subscriptionManager->getSubscriptionCurrentWithRelation($storeOwner);
        if($subscription) {
 
+           $countOrders = $this->getCountOngoingOrders($subscription);
+
+           $subscription['remainingCars'] = $subscription['remainingCars'] - $countOrders;
```

Comment:
Is line 77 correct? I mean you divide orders count of remaining cars and assign the result to remaining cars!!!

Reply:
It wasn't a division, this is a subtraction, yes that's right